### PR TITLE
Feature/release 7.2.2  data grid handle bug

### PR DESCRIPTION
**Pull request by morethanfire**

### CODE REVIEW

**Comment by morethanfire** [comment]
I tested these changes in the react-cm-ui page demo and on HC in the My Ministry roster pages.

File: src/modules/dataGrid.scss [hunk 1]
```diff
@@ -26,18 +26,25 @@ $blackAlpha: rgba(0, 0, 0, .5);
         }
         &_table_component {
             .table-cell:last-child {
-                padding-right: inherit;
+                padding-right: 11px !important;
```
Comment (groberts314) [inline src/modules/dataGrid.scss]: I imagine `!important` is in fact important? ;-)

Comment (groberts314) [reply src/modules/dataGrid.scss]: Do we need a SASS Lint comment?

File: src/modules/dataGrid.scss [hunk 2]
```diff
@@ -26,18 +26,25 @@ $blackAlpha: rgba(0, 0, 0, .5);
...
-                    float: right;
-                    position: relative;
-                    .ui.icon-size-small {
-                        width: 5px;
-                        svg {
+                padding-right: 11px !important;
+                position: relative;
+                .#{$prefixDrawerDataGrid}, .#{$prefixPageDataGrid}  {
+                    &_table_header_handle {
+                        cursor: pointer;
```
Comment (groberts314) [inline src/modules/dataGrid.scss]: Should we consider `cursor: col-resize;`?

Comment (morethanfire) [reply src/modules/dataGrid.scss]: Yep, i like it.

File: src/modules/dataGrid.scss (hunk 1 above)
Comment (morethanfire) [reply src/modules/dataGrid.scss]: Added.

Comment (morethanfire) [reply src/modules/dataGrid.scss]: Yeah, they need to be important because of the old school way of class names taking priority.

Comment (groberts314) [reply src/modules/dataGrid.scss]: Stinkin' legacy CSS.  Got it. 😁

**Pull request merged**